device-types/Ubiquiti/UDM-Pro.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 manufacturer: Ubiquiti
-model: UniFi Dream Machine Pro
+model: Dream Machine Pro
```

Comment:
`UniFi` should be included in the begining as it is the product name. The manufacturer is Ubiquiti and the product name is the `UniFi Dream Machine Pro`